repo2docker/buildpacks/base.py

```diff
@@ -111,7 +111,15 @@
 
 # Allow target path repo is cloned to be configurable
 ARG REPO_DIR=${HOME}
-ENV REPO_DIR ${REPO_DIR}
+ENV REPO_DIR=${REPO_DIR}
+RUN if [ ! -d "${REPO_DIR}" ] \
+    ; then \
+        mkdir -p "${REPO_DIR}" \
+        && chown -R ${NB_USER}:${NB_USER} "${REPO_DIR}" \
+    ; else \
+        echo "${REPO_DIR} already exists..." \
+    ; fi
+
```

Comment:
So, at present, this would fail if you tried to target `/tmp` or `/home` or something, which already exists, but that you didn't (and shouldn't) own.